pontoon/base/urls.py

```diff
@@ -30,6 +30,11 @@
     url(r'^contributor/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
         RedirectView.as_view(url="/contributors/%(email)s/", permanent=True)),
 
+    # API: Toogle user profile attribute
```

Comment:
Do you have any specific design of api in mind? We could use e.g. Piston or something like that.